src/app/shared/components/fy-policy-violation/fy-policy-violation.component.ts

```
@@ -82,8 +82,8 @@ export class FyPolicyViolationComponent implements OnInit {
   }
 
   ngOnInit() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)